src/checks/duplicated_key.rs

```diff
@@ -11,7 +11,7 @@ impl Default for DuplicatedKeyChecker {
     fn default() -> Self {
         Self {
             keys: HashSet::new(),
-            template: String::from("The {} key is duplicated"),
+            template: String::from("DuplicatedKey: The {} key is duplicated"),
```

Comment:
In the future I would like to have the ability to disable some checks by their names.
To do that, all checks should have the `name` field.
Please, move the check's name from the template to the special field.
Also, I would like to have a method to show the check's name in the message, for example:
```rust
impl DuplicatedKeyChecker {
    fn message(&self, key: &str) -> String {
      // code...
    }
}
```

Reply:
I'll implement the message function as specified. Sorry for the delayed response.